plugins/module_utils/net_map/networking_definition.py

```diff
@@ -471,6 +471,26 @@ def check_host_network_ranges_collisions(
     return None, None
 
 
+def _get_field_ip_net_version(raw_field: str) -> typing.Union[int, None]:
```

Comment:
I need to update other usages of Union, but I started using another cleaner approach, `typing.Optional[int]`.

Reply:
Ooh `Optional[X] is equivalent to X | None (or Union[X, None]).` very nice man.  [1]
[1] https://docs.python.org/3/library/typing.html#typing.Optional